package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "playwright-core",
-  "version": "0.14.0-post",
+  "version": "1.0.0-post",
```

Comment:
Is this change intended?

Reply:
Nothing is intended with package-lock :)